atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java

```diff
@@ -93,10 +87,6 @@ public CassandraService(
             Blacklist blacklist,
             CassandraClientPoolMetrics poolMetrics) {
         this.metricsManager = metricsManager;
```

Comment:
I'm wondering if instead you could do @jeremyk-91 's filtering? So filter when this is 1, but publish when it's not?

Reply:
I'm removing it more because I don't think I have ever used it after the initial rollout. So even if we filter it, would it be used in debugging?

Reply:
I've seen it go down and slowly recover before. But I think you might be right, there might not be any in keeping the complexity here